utilities/JointPositionTracking/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+# Authors: Giulio Romualdi
+
+if(FRAMEWORK_COMPILE_YarpImplementation AND FRAMEWORK_COMPILE_Planners)
```

Comment:
Should we consider the compilation of this utility as a separate option?

Reply:
I agree

Reply:
This should go in the direction you suggested  https://github.com/dic-iit/bipedal-locomotion-framework/pull/136/commits/20a7f03eb5bebad24e5f967338db88a67a79ca25